src/Metadata/Driver/AbstractDoctrineTypeDriver.php

```diff
@@ -38,9 +38,13 @@ abstract class AbstractDoctrineTypeDriver implements DriverInterface
         'bigint' => 'integer',
 
         'datetime' => 'DateTime',
+        'datetime_immutable' => 'DateTime',
```

Comment:
Shouldn't this fields be mapped to `DateTimeImmutable` ?

Reply:
Sure. Looks good to me.